corona-warn-app/cwa-app-android
# V2 Configuration Options for Risk Calculation (EXPOSUREAPP-5217)

This PR addresses ExposureApp-5217 (Update Configuration Options for Risk Calculation)

Story:
- changes to the app config may not break older client releases
- the configuration options shall allow to define a mapping of Normalized Time per Day to a Risk Level (already the case)
the risk calculation shall ignored Exposure Windows whose Normalized Time per Day cannot be mapped to a Risk Level 
- the configuration options shall allow to specify the Transmission Risk Value per Transmission Risk Level (one-to-one mapping) instead of using a generic multiplier
-updated protobufs

How to test:
- check if you receive the the new appconfig (v2) inside logcat
- check riskcalculation tests
- check card behavior triggered from testmenu (increased risk ->red, low risk ->green, etc.)

ToDo:

- [x] Replace default app config
